packages/manager/apps/pci-kubernetes/src/api/hooks/useKubernetes.ts

```diff
@@ -39,10 +39,13 @@ export const getAllKubeQueryKey = (projectId: string) => [
   'kube',
 ];
 
-export const useAllKube = (projectId: string) =>
+export const useAllKube = (projectId: string, refetchIntervalTime?: number) =>
```

Comment:
I recommend using options more generically

```suggestion
export const useAllKube = (projectId: string, options: Partial<DefinedOptions...>) =>
```

(We can see together for the exact syntax if you want)